migrations/versions/26c44183abd5_.py

```diff
@@ -0,0 +1,72 @@
+"""empty message
+
+Revision ID: 26c44183abd5
+Revises: 9ea00075af3f
+Create Date: 2019-07-10 22:08:26.666961
+
+"""
+from alembic import op, context
+import sqlalchemy as sa
```

Comment:
please can you run black on this, the auto generated python isn't always formatted correctly.